config/polyfills.js

```diff
@@ -9,8 +7,8 @@ if (typeof Promise === 'undefined') {
 }
 
 if (typeof Set === 'undefined') {
-  window.Set = require('core-js/es6/set');
-  window.Map = require('core-js/es6/map');
+  window.Set = require('core-js/es/set');
+  window.Map = require('core-js/es/map');
```

Comment:
i think we actually don't need those anymore since these were polyfills for ie11.

we're using modern browsers that include these

Reply:
Removed core-js in https://github.com/cfpb/ccdb5-ui/pull/426/commits/606db613e958c3c828f28d71d28a0dbf1d4b0093